vendor/k8s.io/kubernetes/pkg/kubectl/cmd/config/create_cluster.go

```diff
@@ -81,8 +80,6 @@ func NewCmdConfigSetCluster(out io.Writer, configAccess clientcmd.ConfigAccess)
 	options.insecureSkipTLSVerify.Default(false)
 
 	cmd.Flags().Var(&options.server, clientcmd.FlagAPIServer, clientcmd.FlagAPIServer+" for the cluster entry in kubeconfig")
-	cmd.Flags().Var(&options.apiVersion, clientcmd.FlagAPIVersion, "DEPRECATED: "+clientcmd.FlagAPIVersion+" for the cluster entry in kubeconfig")
```

Comment:
when did we deprecate the flag?  You may not be able to remove the altogether.

Reply:
> when did we deprecate the flag? You may not be able to remove the altogether.

Don't respect it, sure.  But if we didn't deprecate in 3.6 or before, flag removal is naughty.